fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/LuceneAnalyzerRegistry.java

```diff
@@ -39,4 +40,9 @@
 public interface LuceneAnalyzerRegistry {
     @Nonnull
     LuceneAnalyzerCombinationProvider getLuceneAnalyzerCombinationProvider(@Nonnull Index index, @Nonnull LuceneAnalyzerType type);
+
+    @Nonnull
+    LuceneAnalyzerCombinationProvider getLuceneAnalyzerCombinationProvider(@Nonnull Index index,
```

Comment:
Should the old overload of this class be removed? It seems like the user should always use this new one to get the right analyzer anyway

Reply:
done.